ls1intum/Artemis
# Feature/attachment upload athene

### Checklist

### Motivation and Context



### Description


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Choose any lecture and add an attachment to it.
4. Check if a request is sent to localhost:8001/upload with the respective file/ 

### Screenshots
